pkgs/by-name/al/alphafold3/package.nix

```diff
@@ -0,0 +1,118 @@
+{
+  lib,
+  fetchFromGitHub,
+  replaceVars,
+  python3,
+  abseil-cpp,
+  dssp,
+  libcifpp,
+  pybind11-abseil,
+  zlib,
+  hmmer,
+}:
+
+let
+  python = python3.override {
+    self = python;
+    packageOverrides = final: prev: {
+      typeguard = prev.typeguard.overridePythonAttrs (oldAttrs: rec {
```

Comment:
If this is for pinning you can relax the dependencies's requirements using `pythonRelaxDeps`

Reply:
I remove the pin and then override it because this package is too old to work.